lib/torque/provider/filterableJson.js

```diff
@@ -53,6 +53,39 @@ var Profiler = require('../profiler');
   filterableJson.prototype = {
 
     /**
+     * Creates a worker to process the tile
+     */
+
+    createProccessTileWorker:function(){
+      var workerFunction = "var proccessTile ="+ this.proccessTileSerial.toString()
+      var wrapper = "; self.onmessage = function(e){var data = JSON.parse(e.data); JSON.stringify(self.postMessage(proccessTile(data.rows,data.coord, data.zoom, data.options)))}"
+      var script = workerFunction + wrapper;
+      var blob = new Blob([script], {type: "text/javascript"})
+      var worker  = new Worker(window.URL.createObjectURL(blob))
+      return worker
+    },
+
+    proccessTile:function(rows,coord,zoom,callback){
+      if(typeof(Worker) === "undefined"){
+        callback(this.proccessTileSerial(rows,coord,zoom, this.options))
+      }
+      else{
+        var worker = this.createProccessTileWorker()
```

Comment:
One thing I noticed creating one worker per request is the threads fight for CPU time, could we limit webworkers to 4 (or some fixed number), kind of a thread pool?